.vscode/launch.json

```
@@ -0,0 +1,21 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Run",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "${workspaceRoot}",
+            "args": ["serve"]
+        },
+        {
+            "name": "Debug",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "${workspaceRoot}",
+            "args": ["serve", "--dev"]
+        },
+    ]
+}
```

```suggestion
{
  "version": "0.2.0",
  "configurations": [
    {
      "name": "Run",
      "type": "go",
      "request": "launch",
      "mode": "auto",
      "program": "${workspaceRoot}",
      "args": ["serve"]
    },
    {
      "name": "Debug",
      "type": "go",
      "request": "launch",
      "mode": "auto",
      "program": "${workspaceRoot}",
      "args": ["serve", "--dev"]
    }
  ]
}

```

Just linting and adding the end of file line break :smile: